README.md

```diff
@@ -27,15 +27,22 @@ You can install QIT in three different ways:
 
 ### Globally Using `wget`
 
-_(Pro Tip: Opting for the Composer installation method simplifies the process of updating QIT in the future 😉)_
+_(Pro Tip: Opting for the Composer installation method simplifies the process of [updating QIT](https://woocommerce.github.io/qit-documentation/#/cli/getting-started?id=updating-qit) in the future 😉)_
 1. Run `wget https://github.com/woocommerce/qit-cli/raw/trunk/qit`
 2. Execute `chmod +x qit`
 3. Move the file to a directory in your PATH, such as `sudo mv qit /usr/local/bin/qit`
 4. Run `qit` to authenticate with your WooCommerce.com Partner Developer account.
 
-## Documentation
+## Examples
 
-For more detailed information on QIT and how to use it, refer to the [official documentation](https://woocommerce.github.io/qit-documentation/#/).
+- `qit run:e2e my-extension` - Runs the WooCommerce Core E2E tests with your extension active.
+- `qit run:e2e my-extension --php_version=8.2` - Same as above, but with PHP 8.2.
+- `qit run:e2e my-extension --php_version=8.2 --woocommerce_version=rc --wordpress_version=rc` - Same as above, but with the release candidate versions of WooCommerce and WordPress, or latest stable if none available.
+- `qit run:e2e my-extension --zip` - Runs the tests using an unreleased development build zip.
+- `qit run:e2e my-extension --additional_wordpress_plugins=gutenberg` - Activates the "Gutenberg" feature plugin in the test environment as well.
+- `qit run:e2e my-extension --optional_features=hpos` - Enables the WooCommerce "High Performance Order Storage" in the test site.
```

Comment:
Minor suggestion, but maybe we can link to HPOS here, also added in the word "feature":

```suggestion
- `qit run:e2e my-extension --optional_features=hpos` - Enables the [WooCommerce "High Performance Order Storage"](https://developer.woocommerce.com/roadmap/high-performance-order-storage/) feature in the test site.
```